### PR TITLE
Initial Release

### PR DESCRIPTION
**Pull request by dtzar**

### CODE REVIEW

**Comment by dtzar** [comment]
@dcaro would appreciate a review :)  IF you do the actual merge, please do a squash merge for a cleaner commit history on master.

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    ssh_keys {
+      path     = "/home/${var.admin_username}/.ssh/authorized_keys"
+      key_data = "${file("${var.ssh_key}")}"
+    }
+  }
+}
+
+resource "azurerm_virtual_machine" "vm-windows" {
+  count = "${contains(list("${var.vm_os_simple}","${var.vm_os_offer}"), "WindowsServer") ? 0 : var.nb_instances}"
```
Comment (dcaro) [inline main.tf]: count = "${contains(list("${var.vm_os_simple}","${var.vm_os_offer}"), "WindowsServer" ) ? var.nb_instances : 0 }"

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+resource "azurerm_resource_group" "vm" {
+  name     = "${var.resource_group_name}"
+  location = "${var.location}"
+  tags     = "${var.tags}"
+}
+
+resource "azurerm_virtual_machine" "vm-linux" {
+  count = "${contains(list("${var.vm_os_simple}","${var.vm_os_offer}"), "WindowsServer") ? var.nb_instances : 0 }"
```
Comment (dcaro) [inline main.tf]: count = "${contains(list("${var.vm_os_simple}","${var.vm_os_offer}"), "WindowsServer" ) ? 0 : var.nb_instances }"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- `vm_os_offer` - The name of the offer of the image that you want to deploy, for example "UbuntuServer" if you are not using the `vm_os_simple` or `vm_os_id` variables. 
+- `vm_os_sku` - The sku of the image that you want to deploy, for example "14.04.2-LTS" if you are not using the `vm_os_simple` or `vm_os_id` variables. 
+- `vm_os_version` - The version of the image that you want to deploy. - default `latest`
+- `public_ip_address_allocation` - Defines how an IP address is assigned. Options are Static or Dynamic. - default `static`
+- `tags` - A map of the tags to use on the resources that are deployed with this module.
+
+Usage
+-----
+
+Provisions 2 Windows 2016 Datacenter Server VMs using `vm_os_simple` to a new VNet and opens up port 22 for SSH access with ~/.ssh/id_rsa.pub :
```
Comment (dcaro) [inline README.md]: should be port 3389 for RDP access.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+```hcl
+  module "mycompute" {
+    source = "github.com/Azure/terraform-azurerm-compute"
+    resource_group_name = "mycompute"
+    location = "East US 2"
+    vm_os_simple = "Windows"
+    public_ip_dns = "mywindowsservers225"
+    remote_port = "3389"
+    nb_instances = 2
+    vnet_subnet_id = "${module.network.vnet_subnets[0]}"
```
Comment (dcaro) [inline README.md]: admin_password = xxxxxx is missing for a successful deployment.

**Pull request merged**